### PR TITLE
Separate k8s version variable for upgrade test

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR separates k8s version variables which are used in upgrade test and anywhere else.
It cleans up KUBERNETES_VERSION and UPGRADED_K8S_VERSION variables from  feature_test_vars.sh

### CODE REVIEW

File: scripts/feature_tests/upgrade/upgrade_vars.sh [hunk 1]
```diff
@@ -18,8 +18,9 @@ export CAPM3_REL_TO_VERSION
 export CAPIRELEASE="v0.3.15"
 export CAPI_REL_TO_VERSION="v0.3.16"
 
-export KUBERNETES_VERSION="v1.21.1"
-export UPGRADED_K8S_VERSION="v1.21.2"
+export FROM_K8S_VERSION="v1.21.1"
+export KUBERNETES_VERSION=${FROM_K8S_VERSION}
+export UPGRADED_K8S_VERSION="v1.21.3"
```
Comment (fmuyassarov) [inline scripts/feature_tests/upgrade/upgrade_vars.sh:23]: Just keep in mind that if you would like to switch to 1.21.3, node images with 1.21.3 will need to be built as well from airship-dev-tools.

File: scripts/feature_tests/upgrade/upgrade_vars.sh [hunk 2]
```diff
@@ -18,7 +18,8 @@ export CAPM3_REL_TO_VERSION
 export CAPIRELEASE="v0.3.15"
 export CAPI_REL_TO_VERSION="v0.3.16"
 
-export KUBERNETES_VERSION="v1.21.1"
+export FROM_K8S_VERSION="v1.21.1"
```
Comment (namnx228) [inline scripts/feature_tests/upgrade/upgrade_vars.sh:21]: What is the meaning of this variable?

**Review by namnx228 — COMMENTED** [review]
/lgtm

**Pull request merged**